### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by rogaldh**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (ivanovaleksey) [inline .editorconfig]: final new line must be here too?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```
Comment (ivanovaleksey) [inline .editorconfig]: I guess it should be 

```
indent_size = 4
...
[.travis.yml]
indent_size = 2
```

Comment (rogaldh) [reply .editorconfig]: Sure

File: .editorconfig (hunk 1 above)
Comment (rogaldh) [reply .editorconfig]: `insert_final_newline = true` works for that

**Comment by ivanovaleksey** [comment]
Thanks!

**Pull request merged**